src/Pkg.jl

```diff
@@ -46,6 +46,7 @@ struct UnstableIO <: IO
 end
 Base.write(io::UnstableIO, b::UInt8) = write(io.io, b)::Int
 Base.get(io::UnstableIO, val, default) = get(io.io, val, default)
+Base.print(io::UnstableIO, arg::Union{SubString{String}, String}) = print(io.io, arg)
```

Comment:
This seems like a method you are not supposed to define as it is not compatible with the usual API for this. It seems like you should make UnstableIO be a subtype of `AbstractPipe`, and get all of the required API functions automatically instead.